relay-server/src/service.rs

```diff
@@ -464,15 +474,33 @@ pub fn create_redis_pools(configs: RedisPoolConfigs) -> Result<RedisPools, Redis
     }
 }
 
+#[cfg(feature = "processing")]
+async fn create_async_pool(config: &RedisConfigRef<'_>) -> Result<AsyncRedisPool, RedisError> {
+    match config {
+        RedisConfigRef::Cluster {
+            cluster_nodes,
+            options,
+        } => AsyncRedisPool::cluster(cluster_nodes.iter().map(|s| s.as_str()), options).await,
+        RedisConfigRef::Single { server, options } => {
+            AsyncRedisPool::single(server.as_str(), options).await
+        }
+        RedisConfigRef::MultiWrite { configs } => {
+            // Based on the assumption that the first config is the
+            // primary config for MultiWrite configurations
+            let primary_config = configs.iter().next().ok_or(RedisError::Configuration)?;
+            Box::pin(create_async_pool(primary_config)).await
```

Comment:
If we don't support it we should return an error.

Reply:
I'm inclined to leave this in because projectconfigs used to work before with a multiwrite config and could now produce errors, especially when using the `Unified` config. So I think technically that would be a breaking change to previous versions.

Reply:
Now it's a breaking change which is silent. The config may not fail, but it does something different.

On that note, this configuration before already did not make sense and was always experimental with a limited lifetime.

Reply:
I don't think it's doing something different to an outside observer. It would always perform the commands for secondary configs in the background ignoring the result. Since it doesn't perform write operations for projectconfigs this just means that less work is done.

> On that note, this configuration before already did not make sense and was always experimental with a limited lifetime.

I see, in this case I agree it is better to return an error and fix the config where needed